src/shared/scrapers/US/CA/glenn-county.js

```diff
@@ -30,7 +30,7 @@ const scraper = {
       $('span:contains("Glenn County COVID-19 Cases")')
         .first()
         .text()
-        .match(/Cases: (\d+)/)[1]
+        .match(/Cases:.*(\d+)/)[1]
```

Comment:
This one got fixed tho? yay!

Reply:
Yup, this was a very small issue that I could fix easily